### PR TITLE
Syncing VstestV3 with VstestV2

### PR DESCRIPTION
**Pull request by virekumar**
**Task name**: Vstest

**Description**: Syncing VstestV3 with VstestV2

**Reverted below commits from VsTestV3**
None

**Added below commits from VsTestV2**
1. Fixing Warning for older version of TestPlat https://github.com/microsoft/azure-pipelines-tasks/commit/b0663cd1885036b7b6ecdb1762b00f2b0d455279
2. Vstest zip uploads with Prod storage account https://github.com/microsoft/azure-pipelines-tasks/commit/652a8ee12ed26308e31b386cdd4dab8db2855ef1

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

image

image

**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Comment by triptijain2112** [comment]
Can you please add testing screenshots?

File: Tasks/VsTestV3/task.json
Comment (adityashahms) [inline Tasks/VsTestV3/task.json:1]: This file is out of sync with VsTestV2. Changes of this PR : https://github.com/microsoft/azure-pipelines-tasks/pull/16772/files#diff-23b0722f458b08abe47ef7330553c77b18139f456b8bfa916eb2b84c1afa353f are not there.

**Comment by adityashahms** [comment]
Please add testing images for the latest iteration since, a lot of files have been changes since yesterday.

File: package-lock.json
Comment (triptijain2112) [inline package-lock.json:1]: We generally don't check in changes to this global package-lock.json file

Comment (virekumar) [reply package-lock.json:1]: File state is same as before it was cut from VstestV2 + changes from VstestV2 after cut + Auto added changes.

File: Tasks/VsTestV3/typings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (triptijain2112) [inline Tasks/VsTestV3/typings.json:1]: What does this file do?
Do we need it?

File: package-lock.json
Comment (virekumar) [reply package-lock.json:1]: These were auto added as part of doing build on src folder for all task. I have removed it now.

File: Tasks/VsTestV3/typings.json (hunk 1 above)
Comment (virekumar) [reply Tasks/VsTestV3/typings.json:1]: Added this file for sync between both versions.

**Review by adityashahms — COMMENTED** [review]
Looks good to me

**Review by triptijain2112 — APPROVED** [review]

**Pull request merged**